templates/csharp/templatedcontrol/NewTemplatedControl.axaml

```diff
@@ -1,16 +1,38 @@
-<Styles xmlns="https://github.com/avaloniaui"
+<ResourceDictionary xmlns="https://github.com/avaloniaui"
         xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
         xmlns:controls="using:AvaloniaAppTemplate.Namespace">
   <Design.PreviewWith>
-    <controls:NewTemplatedControl />
+    
+    <StackPanel Width="400" MinHeight="400">
+       <ThemeVariantScope RequestedThemeVariant="Default">
+          <controls:NewTemplatedControl />      
+       </ThemeVariantScope>
+       <ThemeVariantScope RequestedThemeVariant="Light">
+          <controls:NewTemplatedControl />      
+       </ThemeVariantScope>
+       <ThemeVariantScope RequestedThemeVariant="Dark">
+          <controls:NewTemplatedControl />      
+       </ThemeVariantScope>
+    </StackPanel>
   </Design.PreviewWith>
 
-  <Style Selector="controls|NewTemplatedControl">
-    <!-- Set Defaults -->
+  <ResourceDictionary.ThemeDictionaries>
+      <ResourceDictionary x:Key="Default">
+        <SolidColorBrush x:Key="MyForegroud" Color="Black"/>
+      </ResourceDictionary>
+      <ResourceDictionary x:Key="Light">
+        <SolidColorBrush x:Key="MyForegroud" Color="Black"/>
+      </ResourceDictionary>
+      <ResourceDictionary x:Key="Dark">
+        <SolidColorBrush x:Key="MyForegroud" Color="White"/>
+      </ResourceDictionary>
+  </ResourceDictionary.ThemeDictionaries>
```

Comment:
I am not sure if we should have this in the templates.
I would rather add a comment with a link to the Avalonia.Samples, where we would have this.